### PR TITLE
Minify CSS

### PR DESCRIPTION
**Pull request by dragonx**
This saves about 1MB

### CODE REVIEW

File: cli/cli.ts [hunk 1]
```diff
@@ -993,12 +994,15 @@ function uploadCoreAsync(opts: UploadOptions) {
...
 
                 if (minified) {
-                    const res = uglify.minify(content);
-                    if (!res.error) {
-                        content = res.code;
-                    }
-                    else {
-                        pxt.log(`        Could not minify ${fileName} ${res.error}`)
+                    if (mime == "application/javascript") {
+                        const res = uglify.minify(content);
```
Comment (microsoftsam) [inline cli/cli.ts]: Are there options we can pass here. @dragonx Do you know if uglifycss removes comments as well?

**Comment by dragonx** [comment]
I don't see an option to get rid of comments completely.  The closest I see is this:
`                        content = uglifycss.processString(content, { uglyComments: true });`
That gets rid of newlines in comments, but it doesn't make any significant difference in the file size.  Would you like me to add that?  I have no preference.

**Comment by microsoftsam** [comment]
I think I'd prefer to use the clean-css package. Let me look into this. 
https://github.com/jakubpawlowicz/clean-css

That has support for minifying and removing comments.

**Comment by microsoftsam** [comment]
Closing in favour of #2763. Ended up going with cssnano.

**Pull request closed without merging**